ui/easydiffusion/renderer.py

```diff
@@ -29,8 +29,8 @@ def init(device):
 
 def make_images(req: GenerateImageRequest, task_data: TaskData, data_queue: queue.Queue, task_temp_images: list, step_callback):
     context.stop_processing = False
-    log.info(f'request: {pprint.pformat(get_printable_request(req))}')
-    log.info(f'task data: {pprint.pformat(task_data.dict())}')
+    log.info(f'request: {pprint.pformat(get_printable_request(req).replace("[","\["))}')
+    log.info(f'task data: {pprint.pformat(task_data.dict().replace("[","\["))}')
```

Comment:
Looks like a typo: `task_data.dict().replace`

That's calling replace on the dict. I think you're missing a closing ) before it.